ATM_Selenium_Training/src/test/java/com/epam/atm/selenium/NewApplicationTest.java

```diff
@@ -0,0 +1,74 @@
+package com.epam.atm.selenium;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class NewApplicationTest extends BaseTest {
+
+    @Test
+    public void createNewApplicationWithDefaultSetup() {
+        int x = (int) (Math.random() * 1000);
+        driver.get("https://network-international--qa.cs83.my.salesforce.com/");
```

Comment:
move url login pass and other hardcoded text to constants

Reply:
Done